internal/api/group_test.go

```diff
@@ -240,16 +240,18 @@ func TestCreateGroup(t *testing.T) {
 
 	t.Run("valid", func(t *testing.T) {
 		group, err := client.CreateGroup(&model.CreateGroupRequest{
-			Name:        "name",
-			Description: "description",
-			Version:     "version",
+			Name:          "name",
+			Description:   "description",
+			Version:       "version",
+			MattermostEnv: model.EnvVarMap{"foo": model.EnvVar{Value: "bar"}},
```

Comment:
nit: the upgrade group test seems a little less brittle with the way you set the env var once and used the variable to compare later.